.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  install-dependencies:
+    docker:
+      - image: circleci/node:8
+    working_directory: ~/insight-website
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
```

Comment:
We might need to add `yarn.lock` checksum as well. It can change even if package.json does not.

Reply:
Good catch !